tools/lsp/preview/ui.rs

```diff
@@ -804,6 +804,258 @@ fn update_grouped_properties(
     }
 }
 
+fn get_float(v: &Option<slint_interpreter::Value>) -> Option<f32> {
```

Comment:
For all these function, you can use TryFrom or TryInto instead.
So you'd do `value.try_into().unwrap_or_default()`  or `f32::try_from(value).unwrap_or_default()`

This is valid for all the types you made a getter from.

Reply:
more like `value.clone().and_then(|v| v.try_into().ok())` and then my tests for the conversion start to fail because try_into() is close but not identical.

I was getting tiered to write that.